torch/onnx/symbolic_opset11.py

```diff
@@ -994,3 +996,30 @@ def repeat_interleave(g, self, repeats, dim=None, output_size=None):
     # along the dimension provided, some post-processing is required
     loop_out = g.op("Transpose", loop_out, perm_i=perm_i)
     return reshape(g, loop_out, g.op("Constant", value_t=torch.LongTensor(output_sizes)))
+
+
+def broadcast_tensors(g, self):
+    all_tensors = sym_help._unpack_list(self)
+    final_result = zeros_like(g, all_tensors[0])
+
+    for t in all_tensors:
+        final_result = add(g, final_result, t)
+
+    outputs = g.op("SequenceEmpty", dtype_i=sym_help.cast_pytorch_to_onnx["Float"])
+    for t in all_tensors:
+        outputs = g.op("SequenceInsert", outputs, expand_as(g, t, final_result))
+
+    return outputs
```

Comment:
curious how is broadcast_tensors consumed? i.e. what node takes the output of it as inputs

Reply:
Here is an example of ATEN graph by one of test:

=== ATEN graph:
graph(%0 : Float(1, strides=[1], requires_grad=0, device=cpu),
      %1 : Float(1, strides=[1], requires_grad=0, device=cpu)):
  %6 : Tensor[] = prim::ListConstruct(%0, %1)
  %7 : Tensor[] = aten::broadcast_tensors(%6) # /home/jay/repos/fatcat-z/pytorch/torch/functional.py:72:0
  %x : Float(1, strides=[1], requires_grad=0, device=cpu), %y : Float(1, strides=[1], requires_grad=0, device=cpu) = prim::ListUnpack(%7)
  %30 : Long(requires_grad=0, device=cpu) = prim::Constant[value={0}]()
  %11 : Long(device=cpu) = aten::size(%x, %30) # /home/jay/repos/fatcat-z/pytorch/torch/distributions/normal.py:49:0
  %31 : Long(requires_grad=0, device=cpu) = prim::Constant[value={20000}]()
  %20 : int[] = prim::ListConstruct(%31, %11)
  %32 : Bool(requires_grad=0, device=cpu) = prim::Constant[value={0}]()
  %22 : Float(20000, 1, strides=[0, 1], requires_grad=0, device=cpu) = aten::expand(%x, %20, %32) # /home/jay/repos/fatcat-z/pytorch/torch/distributions/normal.py:64:0
  %33 : Long(requires_grad=0, device=cpu) = prim::Constant[value={20000}]()
  %25 : int[] = prim::ListConstruct(%33, %11)
  %34 : Bool(requires_grad=0, device=cpu) = prim::Constant[value={0}]()
  %27 : Float(20000, 1, strides=[0, 1], requires_grad=0, device=cpu) = aten::expand(%y, %25, %34) # /home/jay/repos/fatcat-z/pytorch/torch/distributions/normal.py:64:0
  %28 : NoneType = prim::Constant()
  %29 : Float(20000, 1, strides=[1, 1], requires_grad=0, device=cpu) = aten::normal(%22, %27, %28) # /home/jay/repos/fatcat-z/pytorch/torch/distributions/normal.py:64:0
  return (%29)

Reply:
i see, i think it would be better to export as `prim::ListConstruct` at the end. If this is possible, the symbolic function can be moved to symbolic_opset9.py and work for all opsets.